src/components/layers-manager/LayersManager.js

```diff
@@ -3,15 +3,15 @@ import React from 'react';
 import LayersForm from './LayersFormContainer';
 import LayersSwitcher from './LayersSwitcherContainer';
 
-function LayersManager() {
+function LayersManager({ isManager }) {
   return(
     <div className="l-layers-manager">
       <div className="row">
         <div className="small-5 columns">
           <LayersForm />
         </div>
         <div className="small-offset-1 small-6 columns">
```

Comment:
Noramlly the use of small is to declare a mobile view (so "small-12 medium-6 medium-offset-1" would make more sense). I know we aren't building for mobile but would be good to cater for smaller screens and those who need to zoom in etc.